### PR TITLE
Document optout.version-check

### PR DESCRIPTION
**Pull request by SeBBBe**
----

If you open a PR that needs to go into a current version, you need to *cherry-pick your commit from dev over to the current version branch*. Only then will the proper builds that generate html/pdf be run. But beware: Docs will be generated but not published automatically!

- [ ] N/A - or - I have added the appropriate "cherry-pick-to" labels to this PR so I don't forget to do this later!

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-ops-manager-238.surge.sh

Updated pages:
- [Server installation](https://neo4j-docs-ops-manager-238.surge.sh/ops-manager/1.10/installation/server)

File: modules/ROOT/pages/installation/server.adoc [hunk 1]
```diff
@@ -325,6 +325,10 @@ For details about configuring proxy in Java, see link:https://docs.oracle.com/en
 | Set to true to opt out of crash analytics being sent to Neo4j (optional)
 | false
 
+| `optout.version-check`
+| `OPTOUT_VERSION_CHECK`
+| Set to true to opt out of automatic fetching of version information for NOM and Neo4j (optional)
```
Comment (alexnb) [inline modules/ROOT/pages/installation/server.adoc:330]:
```suggestion
| Set to true to opt out of automatic fetching of version information for NOM and Neo4j (optional)
| false
```

**Review by alexnb — APPROVED** [review]
LGTM, but needs a default value otherwise the table after that line is corrupted.

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**